contrib/tig-completion.bash

```diff
@@ -93,13 +96,15 @@ if [ -n "$ZSH_VERSION" ]; then
 	bashcompinit
 fi
 
-# we use internal git-completion functions, so wrap _tig for all necessary
+# we use internal git-completion functions (if available), so wrap _tig for all necessary
 # variables (like cword and prev) to be defined
-__git_complete tig _tig 
+if type '__git_complete' 2>/dev/null | grep -q 'function'; then
```

Comment:
(minor) `type -t __git_complete` will output `function`, so you may want to simplify this to:
```suggestion
if [ "$(type -t __git_complete)" = function ]; then
```